src/main/java/controller/planning/EventPlanningController.java

```diff
@@ -92,11 +94,20 @@ private void createUsersInDatabase(List<UserView> listOfUsers) {
         }
     }
 
-    private void createPartitipationsInDatabase(List<Long> users_id, Long party_id){
-        List<Participation> participationsList = new ArrayList<>();
+    private void createParticipationsInDatabase(List<Long> users_id, Long party_id){
+        DBConnectionProvider dbConnectionProvider = new PostgresqlConnectionProvider();
+        try{
```

Comment:
Tries should be merged, and second try should be outside for because if an error happens the rest of operation will be invalid